src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -388,6 +388,12 @@ export class SavedObjectsService
      */
     const skipMigrations = this.config.migration.skip || !pluginsInitialized;
 
+    /**
+     * Note: Prepares all migrations Maps. This will execute the migrations functions
+     * needed for both Individual documents migration and Saved Object migrations on startup.
+     */
```

Comment:
Nit: the comment is a bit misleading and could be interpreted as actually running the migration transform over documents. What about something like...

```suggestion
    /**
     * Note: Prepares all migrations maps. If a saved object type was registered with a `migrations` function, 
     *  this function will be called to get the type's SavedObjectMigrationMap.
     */
```